src/fastoad/notebooks/02_explanation/03_scipy.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p>1.</p><blockquote>The <code>fsolve</code> function will solve the equation</blockquote><p>I suggest an hyperlink to <a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fsolve.html" target="_blank">https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fsolve.html</a></p><p> </p><p> </p><p>2.</p><blockquote>which is similar to how we solve<strong>d</strong> the dichotomy problem</blockquote><p>(just for consistency with the past tense in the next sentence).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Corrected in a03fe54a